src/test/java/org/openrewrite/java/testing/assertj/UseExplicitisEmptyTest.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.testing.assertj;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.DocumentExample;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+@SuppressWarnings("ConstantConditions")
+class UseExplicitisEmptyTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec
+          .parser(JavaParser.fromJavaVersion().classpath("junit-jupiter-api", "assertj-core"))
+          .recipe(new UseExplicitIsEmpty());
+    }
+
+    @Test
+    @DocumentExample
+    void IsEmptyAndIsTrueBecomeIsEmpty() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+                import java.util.Collection;
+                import java.util.ArrayList;
+                
+                import org.junit.jupiter.api.Test;
+                
+                import static org.assertj.core.api.Assertions.assertThat;
+                
+                public class MyTest {
+                
+                    @Test
+                    public void test() {
+                        Collection<String> collection = new ArrayList<>();
+                        collection.add("3");
+                        assertThat(collection.isEmpty()).isTrue();
+                    }
+                }
+              """,
+            """
+                import java.util.Collection;
+                import java.util.ArrayList;
+                
+                import org.junit.jupiter.api.Test;
+                
+                import static org.assertj.core.api.Assertions.assertThat;
+                
+                public class MyTest {
+                
+                    @Test
+                    public void test() {
+                        Collection<String> collection = new ArrayList<>();
+                        collection.add("3");
+                        assertThat(collection).isEmpty();
+                    }
+                }
+              """
+          )
+        );
+    }
+
+    @Test
+    @DocumentExample
```

Comment:
A single example will do; I'm not sure we'd handle two examples well, and it should be clear how the first translates to the second from the description.
```suggestion
```